configs/sgv e-bike study.nrel-op.json

```diff
@@ -0,0 +1,95 @@
+{
+  "url_abbreviation": "sgv e-bike study",
+  "version": 1,
+  "ts": 1741106543,
+  "server": {
+    "connectUrl": "https://sgv e-bike study-openpath.nrel.gov/api/",
+    "aggregate_call_auth": "user_only"
+  },
+  "opcode": {
+    "autogen": false,
+    "subgroups": [
+      "default",
+      "test"
+    ]
```

Comment:
I see, per email from Chris, that you are going to run multiple programs in SGV. I would suggest having different subgroups so you can quickly separate out the programs - the admin dashboard supports filtering by subgroup, for example.